### PR TITLE
Allow to disable default export names

### PR DESCRIPTION
**Pull request by pgrzesik**
Closes: #9361

### CODE REVIEW

File: docs/providers/aws/guide/serverless.yml.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ provider:
       key1: value1
       key2: value2
   deploymentPrefix: serverless # The S3 prefix under which deployed artifacts should be stored. Default is serverless
+  disableDefaultOutputExportNames: true # optional, allows to disable default behavior of generating export names for CloudFormation outputs
```
Comment (medikoo) [inline docs/providers/aws/guide/serverless.yml.md:66]: We should reflect default here, which while being `null`, is better to communicate with `false` (we do like that in other similar cases)

**Review by medikoo — CHANGES_REQUESTED** [review]
@pgrzesik looks great, I've spotted just one documentation issue

File: docs/providers/aws/guide/serverless.yml.md (hunk 1 above)
Comment (pgrzesik) [reply docs/providers/aws/guide/serverless.yml.md]: good suggestion, thank you

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**